datafusion/sql/src/planner.rs

```diff
@@ -2809,6 +2918,7 @@ fn extract_possible_join_keys(
 }
 
 /// Convert SQL simple data type to relational representation of data type
+/*
 pub fn convert_simple_data_type(sql_type: &SQLDataType) -> Result<DataType> {
     match sql_type {
```

Comment:
this is the original function which is now the method of `SqlToRel`

perhaps we should a deprecated notifunction here as this is a pub function

Reply:
I think we should remove it personally or of we want to deprecate it for a release cycle that would be fine

Bonus points for somehow calling the new implementation (like making a dummy `SqlToRel` somehow)